cmd/ops/deprecation.go

```diff
@@ -45,24 +54,28 @@ func deprecationsCmdRun(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Unable to parse schema file %s: %s", schemaFile, err)
 	}
 
-	queriesDir := args[0]
-	queryFields, err := parser.ParseQueryDir(queriesDir, schema)
+	queriesSource := args[0]
+	queryFields, err := parser.ParseQuerySource(queriesSource, schema)
 	if err != nil {
-		return fmt.Errorf("Unable to parse files in %s: %s", queriesDir, err)
+		return fmt.Errorf("Unable to parse files in %s: %s", queriesSource, err)
 	}
 
-	switch outputFormat {
-	case stdoutFormat:
-		deprecationStdOut(queryFields)
-
-	case jsonFormat:
-		err = deprecationJsonOut(queryFields)
-		if err != nil {
-			return err
+	if xcodeFormat {
```

Comment:
If this pass the experimental phase, maybe we can add it as a type of output, which is `plain text` and simplify it a bit so it is `file:line | field | reason` or something like that